build-with-docker.sh

```diff
@@ -0,0 +1,2 @@
+docker run -it --rm --name dynamodb-cross-region-library -v "$PWD":/usr/src/mymaven -w /usr/src/mymaven maven:3.2-jdk-7 mvn clean install
```

Comment:
How will Windows users build and run the image? Perhaps you need to provide a .bat file or add further instructions to users instructing them to run this in the Ubuntu/Linux subsystem for Windows on Windows 10.